### PR TITLE
Update troubleshooting FAQ and small README changes

### PR DESCRIPTION
**Pull request by pedro-f-almeida**
Includes repeated name in group chats problem, clarifies location of messages.hml.

### CODE REVIEW

**Comment by pedro-f-almeida** [comment]
I have a group chat with multiple people, one of which has the same name as I do, and I noticed that there is no way do differentiate between the two of us, unlike there was in the previous formatting, with ID's.
I believe the same happens in direct chat, but I have no evidence.

File: README.rst [hunk 1]
```diff
@@ -308,6 +308,14 @@ seems to be the complete history for each conversation and nothing gets truncate
 
 Unfortunately, this cannot be remedied unless Facebook fixes the problem on their end.
 
+Why are repeated names not showing?
+----------------------------------------
```
Comment (ownaginatious) [inline README.rst]: PyPi's RST parser is extremely sensitive. To ensure this still renders, make sure the number of hyphens matches the number if characters above it.

File: README.rst [hunk 2]
```diff
@@ -308,6 +308,14 @@ seems to be the complete history for each conversation and nothing gets truncate
...
+
+Multiple users with equal names in group chats are shown as a single user. This has to do with Facebook's
+presentation of names in the messages files, which doesn't make this distinction.
```
Comment (ownaginatious) [inline README.rst]: Grammar: message files

**Comment by pedro-f-almeida** [comment]
Alright, I think everything is ready to go.

A bit of a side note, but I don't know whether or not it warrants me creating an issue:
Under `stats.py` shouldn't  `green(" [%d|%.2f%%]\n" % (c, (c * 100) / total)))`
 be changed to ` green(" [%d|%.2f%%]\n" % (c, ((c * 100) / total) + ((c * 100) % total > 0))))`?

Because when stats are printed the percentage only rounds down (as in 15.8% becomes 15%).

**Comment by ownaginatious** [comment]
Great, thanks :+1: Yeah, you're right about the stats thing. I didn't realize it was doing integer division. I wrote that quickly a long time ago and it has pretty much zero test coverage. I was going to do a full rewrite of that later anyway, since others have been requesting things like most frequent words, more than top 10 .etc. I'll fix it then.

**Pull request merged**